src/screens/Favorites/FavoritesPreviewContainer.js

```diff
@@ -5,20 +5,23 @@ import WithAuthentication from '../../util/components/WithAuthentication';
 import { getFilteredCandidates, loadCandidates } from '../../candidate/redux/candidates'
 import { loadUser } from '../../user/redux'
 import { getIsLoggedIn } from '../../auth/selectors';
-import { getUserFavorites } from '../../favorites/redux';
+import { getFavoritesForCategory } from '../../favorites/redux';
 import { Category } from '../../favorites/models';
+var _ = require('lodash');
 
```

Comment:
Maybe someone else can chime in, but I've heard it's better to import the method that you need from lodash so doing `import { flatMap } from 'lodash/flatMap';` (and I guess you'd do the same for uniq) is supposedly better than importing the entire library and keeps the bundle size down.

Reply:
that seems reasonable